### ISSUE TITLE
Wrong destination path?

### ISSUE THREAD

**Issue by krainboltgreene** [synack/docker-rsync#5]
My command: `docker-rsync -watch=false -dst="/usr/src/app" example`
My expectation: `/usr/src/app/{app|lib|config...}`
Actual: `/ust/src/example/{app|lib|config...}`

**Comment by krainboltgreene** [comment]
It doesn't even log correctly?

```
Syncing /Users/krainboltgreene/Code/company/example (local) to /usr/src/app (docker-machine exaplevm)
```

**Comment by mattes** [comment]
I couldn't reproduce this.

**Comment by krainboltgreene** [comment]
I can, from scratch: http://showterm.io/28a8f0e78efe53bce8ff9#387

Here's the truncated version: https://gist.github.com/krainboltgreene/51e0509f5cb10311885e

**Comment by krainboltgreene** [comment]
I'm running based on tag v0.1.1

**Comment by mattes** [comment]
Your output looks fine. Maybe I don't understand the issue. 

```
docker-rsync -watch=false -dst="/usr/src/app" laurelandwolf
Syncing /Users/krainboltgreene/Code/laurelandwolf/decoraid (local) to /usr/src/app (docker-machine laurelandwolf)
```

**Comment by krainboltgreene** [comment]
...Did you read the issue?

I'm specifying a destination of `/usr/src/app`. It's logging `/usr/src/app`. It's **actual destination** is `/usr/src/{{source directory name}}`.

**Comment by mattes** [comment]
Ok, I think I know what you mean now. I was mistaken by the typo `/ust/...` instead of `/usr/...` in your first comment. Sorry about that.

Please add a trailing slash to your dst path: `docker-rsync -watch=false -dst="/usr/src/app/" examplevm`.

**Comment by krainboltgreene** [comment]
Woops, yeah, my bad on that typo.

**Comment by krainboltgreene** [comment]
@mattes Okay, that...doesn't really help. It just namespaces my application to `app/`. When I see a "destination path" I don't assume it means "prefix for destination", I assume it means the absolute path. Presumably that's true for source.

If I'm forced to use the "prefix" version it means I can't make generic docker operations, as every project will have a unique path (`/usr/src/{{project}}` instead of `/usr/src/app`).

**Comment by mattes** [comment]
You're right. Actually I think `-dst` was called `-prefix` before. Couldn't find any traces in the commit log though. 

We should rename `-dst` to `-prefix` and add an actual `-dst` flag that acts as absolute destination path.

See https://github.com/synack/docker-rsync/blob/master/main.go#L56 and https://github.com/synack/docker-rsync/blob/master/rsync.go#L41

Do you want to open a PR?

**Comment by krainboltgreene** [comment]
💚💚💚💚💚💚 Will do, I've never done Go before :3
